configuration/wireless/headless.md

```diff
@@ -6,7 +6,7 @@ Once an image is created on an SD card, by inserting it into a card reader on a
 
 ## Setting up wireless networking
 
-You will need to define a `wpa_supplicant.conf` file for your particular wireless network. Put this file in the `boot` folder, and when the Pi first boots, it will copy that file into the correct location in the Linux root file system and use those settings to start up wireless networking. **After the Pi is connected to power, make sure to wait a few (up to 5) minutes for it to boot up and register on the network.** The Pi's IP address will not be visible immediately after power on, so this step is crucial to connect to it headlessly. Depending on the OS and editor you are creating this on, the file could have incorrect newlines or the wrong file extension so make sure you use an editor that accounts for this. Linux expects the line feed (LF) newline character. For more information, see this [Wikipedia article](https://en.wikipedia.org/wiki/Newline). 
+You will need to define a `wpa_supplicant.conf` file for your particular wireless network. Put this file in the root folder of SD card, and when the Pi first boots, it will copy that file into the correct location in the Linux root file system and use those settings to start up wireless networking. **After the Pi is connected to power, make sure to wait a few (up to 5) minutes for it to boot up and register on the network.** The Pi's IP address will not be visible immediately after power on, so this step is crucial to connect to it headlessly. Depending on the OS and editor you are creating this on, the file could have incorrect newlines or the wrong file extension so make sure you use an editor that accounts for this. Linux expects the line feed (LF) newline character. For more information, see this [Wikipedia article](https://en.wikipedia.org/wiki/Newline). 
```

Comment:
I think talking about "the root folder" would make these instructions even more confusing, but I can see how you've got confused by "the `boot` folder".